WoW-Realm-Tracker/Models/Realm.swift

```diff
@@ -1,27 +1,36 @@
 import Runes
 import Argo
+import Curry
 
 struct Realm {
-    let name: String
-    let type: String
-    let status: Bool
+    enum RealmType: String, CustomStringConvertible {
```

Comment:
Its a protocol which was added in Swift 2 I believe. You can conform to it by having a description property which per the docs:

> This textual representation is used when values are written to an output stream, for example, by print.

So in the use case here, when using it in the label as a string we can just do: `detailTextLabel?.text = "Realm Type \(realm.type)"`